src/history/hs_rec.c

```diff
@@ -912,6 +912,48 @@ __hs_delete_reinsert_from_pos(WT_SESSION_IMPL *session, WT_CURSOR *hs_cursor, ui
         if (__wt_txn_tw_stop_visible_all(session, twp))
             continue;
 
+        /*
+         * The below condition is to handle scenario where the data store and the
+         * history store has same records and ignore.
+         *
+         * Suppose there are two tables table1 and table2 and the below operations are performed.
+         *
+         * 1. Insert a=1 in table1 at timestamp 10
```

Comment:
Do we need table to for this example?

Reply:
Yeah, it can be simplified with just one table but to make it more realistic where the data store has more content than the history store table.

Reply:
I simplified the example with just one table.